coderd/externalauth/externalauth.go

```diff
@@ -28,6 +29,13 @@ import (
 	"github.com/coder/retry"
 )
 
+const (
+	// failureReasonLimit is the maximum text length of an error to be cached to the
+	// database for a failed refresh token. In rare cases, the error could be a large
+	// HTML payload.
+	failureReasonLimit = 200
```

Comment:
200 characters isn't very long. for example, this comment is exactly 200 characters long, and contains a decent amount of information, but not an incredible amount. it's probably ok, but may be short.